packages/gatsby/src/commands/develop-process.ts

```diff
@@ -248,6 +248,10 @@ async function startServer(program: IDevelopArgs): Promise<IServer> {
     activity.start()
     await sourceNodes({
       webhookBody: req.body,
```

Comment:
should we consider this deprecated in favour of `webhookRequest.body`?

Reply:
Probably but let's first get this in cause a message could get spooky